jazkarta/shop/browser/templates/shipping_form.pt

```diff
@@ -109,58 +109,115 @@
       <tal:actions metal:fill-slot="actions" />
     </tal:form>
 
-    <script>
-    jQuery(function($) {
-      var $form = $('#form');
-      var $street = $('input[name="form.widgets.street"]');
-      var $city = $('input[name="form.widgets.city"]');
-      var $state_text = $('input[name="form.widgets.state"]');
-      var $state_select = $('#state-select');
-      var $prov_select = $('#province-select');
-      var $zip = $('input[name="form.widgets.postal_code"]');
-      var $country = $('select[name="form.widgets.country"]');
+    <tal:using_p5 tal:condition="view/using_plone5">
```

Comment:
I have an idea for a different way to handle this (not sure if better or not): We could put the p4 and p5 javascript in separate files in two different resource directories, and register them with the same name but using conditional ZCML so it registers the right one for the version of Plone. The only reason it's inline javascript is because I was lazy.